### PR TITLE
Create security_baseline_sandbox_stage.md

### PR DESCRIPTION
**Pull request by Danajoyluck**

### CODE REVIEW

**Review by sevansdell — CHANGES_REQUESTED** [review]
I think this is coming along well. There are three places I didn't see information before I'm comfortable approving.

1. In the goal section, can you describe a bit more about what the SIG will do to evolve the baseline to prepare for LF wide adoption? (e.g. work with graduated projects to proof of concept the baseline and get feedback). What does "good" look like to know you're done and ready to apply to rest of LF?
2. in the governance section, please clarify the initial governance request (if I remember correctly from the TAC meeting your request is to initially fall under TAC governance.
3.  URLS. I believe there was some conversation about if the initial repo should be in the TAC, or a separate repo while evaluating the baseline for OpenSSF. In some comments I understood this might evolve when it applies to the rest of the LF. Need to decide and populate initial URLs.

Thanks!

**Comment by Danajoyluck** [comment]
> I think this is coming along well. There are three places I didn't see information before I'm comfortable approving.
> 
> 1. In the goal section, can you describe a bit more about what the SIG will do to evolve the baseline to prepare for LF wide adoption? (e.g. work with graduated projects to proof of concept the baseline and get feedback). What does "good" look like to know you're done and ready to apply to rest of LF?
> 2. in the governance section, please clarify the initial governance request (if I remember correctly from the TAC meeting your request is to initially fall under TAC governance.
> 3. URLS. I believe there was some conversation about if the initial repo should be in the TAC, or a separate repo while evaluating the baseline for OpenSSF. In some comments I understood this might evolve when it applies to the rest of the LF. Need to decide and populate initial URLs.
> 
> Thanks!

Thanks a lot @sevansdell I updated the document to address question 1 and 2. I assumed 3 is comment. Once the SIG is formed, we will have a repo,  the URIs and document update process  will be sorted out with TAC

File: process/sig-lifecycle-documents/security_baseline_sandbox_stage.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  * Adolfo "Puerco" García Veytia, CNCF kubernetes SIG Release Technical Lead, OpenSSF Protobom, OpenVEX maintainer, Staklock, GitHub ID: puerco
+  * Justin Cappos, CNCG TUF, in-toto, Uptane, OpenSSF gittuf maintainer, New York University.  GitHUb ID: JustinCappos
+  * David Wheeler, OpenSSF Best Practice Badge maintainer, OpenSSF, GitHub ID: david-a-wheeler
+  * Dana Wang, OpenSSF security baseline maintainer, OpenSSF, GitHub ID: danajoyluck
+
+### Governing Body
+SIGs may report to an existing OpenSSF Working Group or directly to the TAC as their governing body. The SIG commits to providing the governing body quarterly updates on progress.
+  * Security Best Practices Working Group
+
+CRob and Dana Wang had conversations about this inititve. CRob has agreed to be the sponsor of this SIG and welcome the group to join Security Best Practices Working Group.
```
Comment (lehors) [inline process/sig-lifecycle-documents/security_baseline_sandbox_stage.md:30]:
```suggestion
CRob and Dana Wang had conversations about this initiative. CRob has agreed to be the sponsor of this SIG and welcome the group to join Security Best Practices Working Group. 
```

**Review by marcelamelara — APPROVED** [review]
Thanks @Danajoyluck ! This looks good to me!

**Review by SecurityCRob — APPROVED** [review]
The BEST WG voted to adopt Baseline as a SIG

**Review by mlieberman85 — APPROVED** [review]
lgtm

**Review by sevansdell — APPROVED** [review]
Thanks for reviewed changes.

**Review by camaleon2016 — APPROVED** [review]

**Review by steiza — APPROVED** [review]

**Review by torgo — APPROVED** [review]
lgtm 👍🏻

**Review by marcelamelara — APPROVED** [review]
Not sure why my original approval was removed, but I still approve this!

**Pull request merged**